lib/SubscriptionsCheckout.php

```diff
@@ -9,7 +9,7 @@ abstract class SubscriptionsCheckout
   const SANDBOX_BASE_URL = "https://sandbox.api.getsafepay.com";
   const PROD_BASE_URL = "https://getsafepay.com";
 
-  const REQUIRED_OPTIONS = ['environment', 'plan_id', 'cancel_url', 'redirect_url', 'tbt'];
+  const REQUIRED_OPTIONS = ['environment', 'plan_id', 'reference', 'cancel_url', 'redirect_url', 'tbt'];
```

Comment:
Should this be required or optional?

Reply:
Although it is optional on the API level but decided to make it required for subscription checkout URL. It will overcome that different email signup issue.

Reply:
Have made it optional as per internal discussion.